RecoEgamma/ElectronIdentification/test/eidSelectorTest_cfg.py

```diff
@@ -4,7 +4,7 @@
 
 process.load('FWCore/MessageService/MessageLogger_cfi')
```

Comment:
Could you please also change
```suggestion
process.load('FWCore.MessageService.MessageLogger_cfi')
```